### ISSUE TITLE
Class Solspace\Freeform\controllers\notifications\SenderController [...] does not comply with psr-4 autoloading standard. Skipping.

### ISSUE THREAD

**Issue by darylknight** [solspace/craft-freeform#393]
**Describe the bug or issue you're experiencing**
Updating Freeform to the latest version (from 4.0.0-beta.12) is showing errors during the upgrade process.

**Steps to reproduce**

1. Run `composer update`
2. Errors below appear after the "Generating optimized autoload files" step

```
  - Upgrading solspace/craft-freeform (4.0.0-beta.12 => 4.0.0-beta.15): Extracting archive
Generating optimized autoload files
Class Solspace\Freeform\controllers\notifications\SenderController located in ./vendor/solspace/craft-freeform/packages/plugin/src/Controllers/notifications/SenderController.php does not comply with psr-4 autoloading standard. Skipping.
Class Solspace\Freeform\controllers\notifications\DatabaseController located in ./vendor/solspace/craft-freeform/packages/plugin/src/Controllers/notifications/DatabaseController.php does not comply with psr-4 autoloading standard. Skipping.
Class Solspace\Freeform\controllers\notifications\AbstractNotificationsController located in ./vendor/solspace/craft-freeform/packages/plugin/src/Controllers/notifications/AbstractNotificationsController.php does not comply with psr-4 autoloading standard. Skipping.
Class Solspace\Freeform\controllers\notifications\FilesController located in ./vendor/solspace/craft-freeform/packages/plugin/src/Controllers/notifications/FilesController.php does not comply with psr-4 autoloading standard. Skipping.
```

**Craft & Plugin Info (please complete the following information):**

- Craft Version: 4.0.5
- Freeform Version: 4.0.0-beta.12
- Freeform Edition: Pro
- Fresh Install or Upgrade: Upgrade

**Comment by darylknight** [comment]
Looks like this might be related to Beta 14 from https://github.com/solspace/craft-freeform/discussions/374. When I go to /admin/freeform/notifications on the live site, I get this: 

![image](https://user-images.githubusercontent.com/1250017/178911845-4a4bd4d5-d093-4ab3-9d05-17cef86df68d.png)

**Comment by kjmartens** [comment]
Sorry about that @darylknight. We've fixed this in v4.0.0-beta.15.1 now. 🙂
